### PR TITLE
Issue #3979 Fixed

### PR DESCRIPTION
**Pull request by Nikitha2309**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes , I have clearly read the guidelines

### Description

Given a number , Find whether it is even or odd without using Arithmetic operators in CPP

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#3979

### CODE REVIEW

File: C-Plus-Plus/math/Find_Even_Or_Odd_without_arithmetic_operators.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*
+Author - Nikitha Reddy [Github ID - Nikitha2309]
```
Comment (rohanbabbar04) [inline C-Plus-Plus/math/Find_Even_Or_Odd_without_arithmetic_operators.cpp]: Remove this line

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]
No need to comment before each and every line only important ones

File: C-Plus-Plus/math/Find_Even_Or_Odd_without_arithmetic_operators.cpp (hunk 1 above)
Comment (Nikitha2309) [reply C-Plus-Plus/math/Find_Even_Or_Odd_without_arithmetic_operators.cpp]: Done , Author Name removed from the code

**Comment by Nikitha2309** [comment]
> No need to comment before each and every line only important ones

Okay , I will note this point and take care from next time .

**Comment by Nikitha2309** [comment]
@rohanbabbar04 @HarshCasper 
will u merge the code now ?
Are there any other changes I have to make?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
It doesn't fall under the scope of a DS-Algo repository.

**Pull request closed without merging**